.github/scripts/test-project.sh

```diff
@@ -28,6 +28,13 @@ echo ""
 echo "-----------------------------"
 echo "running $dir/$project"
 
+echo "cd $dir/$project"
+cd "$dir/$project"
+
+PM=$(node -e "require('@antfu/ni').detect({ autoInstall: false, cwd, programmatic: true }).then(console.log)")
+INSTALL_CMD=$(node -e "require('@antfu/ni').getCommand('$PM', 'install', []).then(console.log)")
+
+sh -c "$INSTALL_CMD"
```

Comment:
Prepares the project before had, so it can use the installed cli version of the project.